### PR TITLE
Added two lambda functions within product-service

### PR DESCRIPTION
**Pull request by Spiralfate**
FE repo PR: https://github.com/Spiralfate/nodejs-aws-fe/pull/2

products list endpoint: https://6z1ek1l58b.execute-api.us-east-1.amazonaws.com/dev/products
products by id endpoint:
 - https://6z1ek1l58b.execute-api.us-east-1.amazonaws.com/dev/product?id=7567ec4b-b10c-48c5-9345-fc73c48a80a2
 - https://6z1ek1l58b.execute-api.us-east-1.amazonaws.com/dev/product?id=7567ec4b-b10c-48c5-9345-fc73c48a80a1
 - https://6z1ek1l58b.execute-api.us-east-1.amazonaws.com/dev/product?id=7567ec4b-b10c-45c5-9345-fc73c48a80a1

### CODE REVIEW

**Comment by Max-Korsakov** [comment]
Please, add readme with description of what points were done.

**Comment by Max-Korsakov** [comment]
FE link doesn't provided (3 points from 4)
Async/await +
Modules +
Webpack +
Swagger -
Unit tests -
Different files +
Error handling - 

Overall - 7 points

File: product-service/serverless.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        }
+      ]
+    },
+    product: {
+      handler: 'handler.getProductsById',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'product',
```
Comment (maryna-karpovets) [inline product-service/serverless.ts]: Why don't you use path params?
```suggestion
            path: 'product/{id}',
```

File: product-service/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (maryna-karpovets) [inline product-service/.vscode/launch.json:1]: I believe it should be in `.gitignore`

File: product-service/functions/getProductsById.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+const fetchProductById = async (id) => {
+    const products = await Promise.resolve(productsList);
+    const product = products.find(product => product.id === id);
+
+  return product;
+}
+
+const getProductById: APIGatewayProxyHandler = async event => {
+    const product = await fetchProductById(event.queryStringParameters.id);
```
Comment (maryna-karpovets) [inline product-service/functions/getProductsById.ts]: You can use `pathParameters.id` if you get the `id` from the path parameters. Usually, it is passed not like a query but like path param.

**Pull request merged**